### DISCUSSION TITLE
Laughter for Aura Text-to-Speech

### DISCUSSION THREAD

**Discussion by chromagic-development** [deepgram/community#885] [Feature Requests]
Is there currently a way to introduce laughter for Aura voices or is this a Feature Request? I would like to explore generating laughter with the voice models after classifying or creating humor in a conversation.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@chromagic-development  There isn't a way to do that yet with our TTS Models, but I did convert this to a Feature Request. What are you trying to do? I am curious. 😄

**Reply by chromagic-development** [reply]
I have been tinkering with a voice assistant project that also integrates a LLM (currently, ChatGPT) and I would like the option to emote laughter during conversations. Your excellent TTS models are already capable of vocalizations such as “um” and pauses, so this feature would simply add another option to the repertoire for natural sounding speech interactions.

**Comment by jpvajda** [comment]
I'm adding Our PM of TTS to this thread. @michellelychan
